litellm/tests/test_promptlayer_integration.py

```diff
@@ -65,8 +64,33 @@ def test_promptlayer_logging_with_metadata():
         print(e)
 
 
-test_promptlayer_logging_with_metadata()
+def test_promptlayer_logging_with_metadata_tags():
+    try:
+        # Redirect stdout
+        old_stdout = sys.stdout
+        sys.stdout = new_stdout = io.StringIO()
+
+        response = completion(
+            model="gpt-3.5-turbo",
+            messages=[{"role": "user", "content": "Hi 👋 - i'm ai21"}],
+            temperature=0.2,
+            max_tokens=20,
+            metadata={"model": "ai21", "pl_tags": ["env:dev"]},
+        )
```

Comment:
LGTM, can you set `mock_response="some text"` so it doesn't make the OpenAI API call for this test